src/Chapter1.hs

```diff
@@ -653,8 +671,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Integral t => t -> t
+firstDigit n
+  | abs n `div` 10 == 0 = abs n
+  | otherwise = firstDigit (abs n `div` 10)
```

Comment:
Awesome solution! 💯 

You see that you use `abs n` in all of the cases, so you could move it into a separate value and use it everywhere 😉